invenio_app_ils/circulation/mail/tasks.py

```diff
@@ -37,3 +44,39 @@ def send_ils_mail(factory, prev_loan, loan, trigger, **kwargs):
     ))
     data = msg.__dict__
     send_email.apply_async((data,), link=log_successful_mail.s(data))
+
+
+def send_overdue_mail(loan, **kwargs):
+    """Send loan overdue email message async and log the result in Celery.
+
+    :param loan: the overdue loan.
+    """
+    factory = overdue_loan_message_factory()
```

Comment:
I don't see the recipients, but maybe I am missing something...
Can you also please ensure that for any e-mail sent, if not PROD, recipients are sent to a new config `MAIL_NOTIFY_TEST_RECIPIENTS` or similar?